hackathon_site/dashboard/frontend/src/components/team/TeamCardAdmin/TeamCardAdmin.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styles from "./TeamCardAdmin.module.scss";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Card from "@material-ui/core/Card";
+
+interface CardProps {
+    teamCode: string;
+    members: string[];
+}
+
+export const TeamCardAdmin = ({ teamCode, members }: CardProps) => {
+    const title = `Team ${teamCode}`;
+
+    return (
+        <Card style={{ minHeight: "200px" }}>
+            <Container className={styles.content_container}>
+                <Container className={styles.title}>
+                    <Typography variant={"h6"}>{title}</Typography>
+                </Container>
+                {members?.length
+                    ? members?.map((member, i) => (
+                          <Container className={styles.name} key={i}>
+                              <Typography variant="body2" noWrap>
```

Comment:
I think the noWrap gets a bit problematic when the team names are longer than expected, the grid starts to look weird. Could you remove it
![image](https://user-images.githubusercontent.com/20618460/172745779-40e03a73-950a-4aae-843a-c2d22cbda98e.png)

Reply:
removed!